apps/protocol-frontend/src/components/BulkAttestationModal.tsx

```diff
@@ -11,12 +11,12 @@ import {
 } from '@chakra-ui/react';
 import { useUser } from '../contexts/UserContext';
 import { MdCheckCircle } from 'react-icons/all';
-import { MintContributionType } from '../types/mint';
 import { useContributions } from '../contexts/ContributionContext';
+import { AttestationTableType } from '../types/table';
 import pluralize from 'pluralize';
 
 interface BulkAttestationModalProps {
-  contributions: MintContributionType[];
+  contributions: AttestationTableType[];
```

Comment:
Love this -- really great idea to move these into their own file so we can reuse